packages/experiments/src/components/fluent/theme/FluentTheme.ts

```diff
@@ -0,0 +1,22 @@
+import { createTheme, ITheme } from "office-ui-fabric-react";
+import { FluentColors } from "@uifabric/experiments/lib/components/fluent/theme/FluentColors";
+
+const FluentTheme: ITheme = createTheme({
+  palette: {
```

Comment:
This will be expanded over time, as all of the current colors (not just grays) have equivalents in the Fluent palette.